devices/bosch.js

```diff
@@ -12,6 +12,26 @@ const ea = exposes.access;
 // Radiator Thermostat II
 const boschManufacturer = {manufacturerCode: 0x1209};
 
+
+//BMCT-SLZ
+const stateDeviceType = {
+  'Light': 0x04,
```

Comment:
Please make all the values here snake case, e.g.

- `Light` -> `light`
- `Rocker Switch - Key Change` -> `rocker_switch_key_change`